src/main/scala/Main.scala

```diff
@@ -1,3 +1,28 @@
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.server.Directives._
+
+import scala.concurrent.ExecutionContext
+import scala.io.StdIn
+
 object Main extends App {
-  println("Hello World!")
+  private implicit val system: ActorSystem[Nothing] = ActorSystem(Behaviors.empty, "main-system")
+  protected implicit val executor: ExecutionContext = system.executionContext
+
+  val route =
+    path("health") {
+      get {
+        complete(StatusCodes.OK, HttpEntity(ContentTypes.`application/json`, "200"))
```

Comment:
You don't need to return `"200"`. And also content type should be set by default.